components/pcloud/props-custom-descriptions.mjs

```diff
@@ -0,0 +1,40 @@
+import pcloud from "./pcloud.app.mjs";
```

Comment:
Thanks for the feedback! Perhaps because I modified the props and their descriptions several times after importing them, and wanted to make sure they were as clear as possible to the user, I ended up going too deep into avoiding duplicating these values.

I agree this method of reusing the props doesn't feel very clear, and am transferring them to the propDefinitions as per your suggestion,